src/components/custom/_signup-affiliate.js

```diff
@@ -0,0 +1,217 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { navigate } from 'gatsby'
+import AgreementLabel from './_agreement-label'
+import { Input, Button } from 'components/form'
+import { Header, Text, LinkText } from 'components/elements'
+import { localize } from 'components/localization'
+import device from 'themes/device.js'
+
+const SignupContent = styled.div`
+    width: 48.4rem;
+    display: flex;
+    justify-content: flex-start;
+    flex-direction: column;
+    padding: 4rem;
+    background-color: var(--color-white);
+    border-radius: 0.6rem;
+    box-shadow: 0 1.6rem 2rem 0 rgba(0, 0, 0, 0.1);
+
+    @media ${device.tablet} {
+        width: 53rem;
+    }
+    @media ${device.mobileL} {
+        width: 100%;
+        padding: 6rem 2rem;
+    }
+`
+const StyledHeader = styled(Header)`
+    @media ${device.tabletL} {
+        font-size: 3rem;
+        margin-bottom: 3rem;
+    }
+`
+
+const SubTitle = styled(Text)`
+    @media ${device.tabletL} {
+        font-size: 2rem;
+        margin-bottom: 1rem;
+    }
+`
+const StyledText = styled(Text)`
+    @media ${(props) => device.tabletL && props.notedBox} {
+        width: 13rem;
+    }
+    @media (max-width: 340px) {
+        width: 17rem;
+    }
+    @media ${device.tabletL} {
+        font-size: ${(props) => props.tabletFontSize || 'var(--text-size-xxs)'};
+    }
+`
```

Comment:
You can replace this whole block with the use of `Header` with the correct type required. Instead of these media queries, you can use responsive props such as `tabletL` and pass in CSS style object for styling. Another note is that `var(--text-size-xxs)` is deprecated and should no longer be used.